types/src/keyless/test_utils.rs

```diff
@@ -444,9 +444,7 @@ mod test {
         public_inputs_hash: [u8; 32],
     }
 
-    // Run the prover service locally - https://github.com/aptos-labs/prover-service
-    // Then run ./scripts/dev_setup.sh
-    // Lastly run ./scripts/run_test_server.sh
+    // Run the prover service locally at http://localhost:8083/
```

Comment:
Can you add back the new URL to the prover service repo with the right commands to run it locally **or** a link to a README that shows how to run it locally? 🙏